samples/Commerce/Commerce.UI/App.xaml

```diff
@@ -23,18 +23,18 @@
 				<ToolkitResources xmlns="using:Uno.Toolkit.UI" />
 
 				<!-- Load Material Toolkit resources -->
-				<MaterialToolkitResourcesV1 xmlns="using:Uno.Toolkit.UI.Material" />
+				<MaterialToolkitResources xmlns="using:Uno.Toolkit.UI.Material" />
```

Comment:
Also, we'll need to edit the ColorPaletteOverride.xaml to conform to the new color keys

Reply:
Yes, but for that we'll take the new colors from the Figma document, which is not ready yet.